doc/CHANGELOG.rst

```diff
@@ -3,50 +3,53 @@ Changelog
 
 This changelog *only* contains changes from the *first* pypi release (0.5.4.3) onwards.
 
-- **Next version - unreleased**
+- **0.6.4 - unreleased**
+  - Added extra verbiage to the to the raised exception when an overloaded
+    method could not be matched.  It now prints a list of all possible method
+    signatures.
```

Comment:
Sorry for the many changes in this file.  This block with this comment is the only real addition.  The other changes are from trailing whitespaces that got removed.

Reply:
please keep the "next version" string, since I think that the bumpversion tool looks for this when we make a release.

Reply:
Oh I see.  I reverted that line.